src/test/kotlin/org/objectionary/ddr/TestBase.kt

```diff
@@ -73,8 +83,30 @@ interface TestBase {
 
     /**
      * @return name of the test being executed
+     * @throws TestNotFoundException
+     */
+    fun getTestName(): String {
+        Thread.currentThread().stackTrace.forEach {
+            val methodName = it.methodName
+            if (methodName.substring(0, "test ".length) == "test ") {
+                return methodName.substring("test ".length).replace(' ', '_')
+            }
+        }
+        throw TestNotFoundException("Function with a name starting with \"test\" is not found in the current stack trace")
+    }
+
+    /**
+     * Deletes temporary output directory
+     *
+     * @param pathToSource path to source directory
+     * @throws IOException
      */
-    fun getTestName() = Thread.currentThread().stackTrace[4].methodName
-        .substring(5)
-        .replace(' ', '_')
+    fun deleteTempDir(pathToSource: Path) {
+        val tmpDir = File("${pathToSource}_$postfix")
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html